packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/breadcrumbs.twig

```diff
@@ -1,14 +1,23 @@
-
-<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
-  <nav class="ma__breadcrumbs__container">
-    <div class="ma__breadcrumbs__label">
-      More in:
-    </div>
-    <ol class="ma__breadcrumbs__items">
-      <li class="ma__breadcrumbs__item"><a href="/">Home</a></li>
-      {% for link in breadcrumbs.links %}
-        <li class="ma__breadcrumbs__item"><a href="{{link.href}}">{{link.text}}</a></li>
-      {% endfor %}
-    </ol>
-  </nav>
-</section>
+<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
+  <nav aria-label="Breadcrumbs" class="ma__breadcrumbs__container">
+    <ol class="ma__breadcrumbs__items">
+      {% if breadcrumbs.links|length <= 2 %}
+        <li class="ma__breadcrumbs__item ma__breadcrumbs__chevron-back" aria-hidden="true"></li>
+      {% endif %}
+      {% for link in breadcrumbs.links %}
+        {% if link.url %}
+          <li class="ma__breadcrumbs__item" id="ma__breadcrumbs__item--{{ loop.index }}">
+            {% if loop.first %}
+              <a href="{{link.url}}" aria-label="Home" class="ma__breadcrumbs__item__icon-link">{{ icon("home") }}</a>
+            {% else %}
+              <a href="{{link.url}}">{{link.text}}</a>
+            {% endif %}
+          </li>
+        {% endif %}
+      {% endfor %}
+      <li class="ma__breadcrumbs__item ma__breadcrumbs__item__expand-indicators">
+        <button type="button" class="js-breadcrumbs-button" aria-label="Show all Breadcrumbs" title="Show all Breadcrumbs" aria-pressed="false">…</button>
```

Comment:
```suggestion
        <button type="button" class="js-breadcrumbs-button" aria-label="Show all Breadcrumbs" title="Show all Breadcrumbs" aria-pressed="false">…</button>
```
aria-pressed is not changing value. I also wonder if it's necessary, as it's not a true toggle button. 
![Screen Shot 2021-12-02 at 10 11 22 AM](https://user-images.githubusercontent.com/5789411/144449007-2dd53e10-97ad-4a82-960b-7f1fede765b6.png)

Reply:
fixed